model/survey-response-logger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @module model/survey-response-logger
+ */
+
+const Sequelize = require('sequelize');
+
+/**
+ * Registers model with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * Each SurveyResponseLogger will be associated with a completed SurveyInstance.
+     * Each record represents the timestamps associated with when the question was answered.
+     * It also holds the timestamps for next and previous buttons pressed for each question while filling the survey.
+     * @typedef {Object} SurveyResponseLogger
+     */
+    sequelize.define(
+        'survey_response_logger',
+        {
+            ansTimestamp: {
+                type: Sequelize.DATE
+            },
+            prevTimeStamp: {
+                type: Sequelize.DATE
+            },
+            nextTimeStamp: {
+                type: Sequelize.DATE
+            }
```

Comment:
Okay

Could you document this
> a. ansTimestamp - this is the timestamp when a specific option was chosen by the user for answering the question
> b. prevTimestamp - is the timestamp when the previous button was clicked for that question 
> c. nextTimestamp - is the timestamp when the next button was clicked for the respective question.

Using `@property` definitions in the `@typedef`

E.G.
https://github.com/promisasu/Portal/blob/master/model/question-template.js#L17-L19